cmd/poseidon/poseidon.go

```diff
@@ -95,44 +67,37 @@ func schedule(fc firmament.FirmamentSchedulerClient) {
 			}
 		}
 		// TODO(ionel): Temporary sleep statement because we currently call the scheduler even if there's no work do to.
-		time.Sleep(time.Duration(schedulingInterval) * time.Second)
+		time.Sleep(time.Duration(config.GetSchedulingInterval()) * time.Second)
 	}
 }
 
-func WaitForFirmamentSerive(fc firmament.FirmamentSchedulerClient) {
+// WaitForFirmamentService blocks till the Firmament service is available
+func WaitForFirmamentService(fc firmament.FirmamentSchedulerClient) {
 
-	service_req := new(firmament.HealthCheckRequest)
+	serviceReq := new(firmament.HealthCheckRequest)
 	err := wait.PollImmediate(2*time.Second, 10*time.Minute, func() (bool, error) {
-		ok, _ := firmament.Check(fc, service_req)
+		ok, _ := firmament.Check(fc, serviceReq)
 		if !ok {
 			return false, nil
 		}
 		return true, nil
 	})
 	if err != nil {
-		glog.Fatalf("Timedout waiting for firmament service %v", err)
+		glog.Fatalf("Timed-out waiting for firmament service %v", err)
 	}
 }
 
 func main() {
-	glog.Info("Starting Poseidon...", firmamentAddress)
-	fc, conn, err := firmament.New(firmamentAddress)
+	glog.Info("Starting Poseidon...", config.GetFirmamentAddress())
```

Comment:
why not initialize `poseidonConfig` in pkg config and then call its methods? `config` is usually leveraged for a variable name.

Reply:
I don't want to move it from init methods, because of the command line flag are best parsed at the init function.
I have kept the variable for ```poseidonConfig``` as a global in the config package, beacuse it will be private to the package and cannot be accidentially modified by other methods otherside the package.
If we remove the command line flag and use the config flag only, then we can remove it from init method. But it will good idea to keep the config variable private to the config package.